modules/opp-architecture-architecture.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * architecture/opp-architecture.adoc
+
+:_module-type: CONCEPT
+[id="opp-architecture-architecture_{context}"]
+= {product-title} description and architecture
+
+{product-title} builds on the capabilities of {ocp} with the following features:
+
+* Multi-cluster security
+* Complete management capabilities
+* Integrated data management
+* A global container registry
+
+{product-title} protects and manages applications across open hybrid cloud environments and application lifecycles.
+
+image::242_OpenShift_Plus_0422.png[]
+
+{product-title} supports these additional capabilities:
+
+* Platform services
+** Service mesh, serverless
+** Builds, CI/CD pipelines
+** GitOps, Distributed tracing
+** Log management
+** Cost management
+** Vulnerability management
+** compliance
+
+* Application services
+** Languages and runtimes
+** API management
+** Integration
+** Messaging
+** Process automation
+
+* Data services
+** Databases, cache
```

Comment:
Do we want a single coma here? Maybe _and_? Not sure what kind of cache this means.